udata_youckan.py

```diff
@@ -106,7 +106,7 @@ def authorized():
     response = youckan.get('me')
     data = response.data
 
-    user = datastore.find_user(slug=data['slug'])  # TODO: use user id instead
+    user = datastore.find_user(email=data['email'])
```

Comment:
Yes, but we don't have this information right now. This change only ensure more robust fetch and not duplicating users